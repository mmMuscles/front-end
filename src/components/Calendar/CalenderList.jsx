import React from "react";

const themeList = [
  "Rest",
  "Abs",
  "Arms",
  "Back",
  "Calves",
  "Chest",
  "Legs",
  "Shoulders",
  "Yoga",
  "Stretch",
];

export default function CalenderList({ handleChange, todaysTheme }) {
  return (
    <select
      aria-label="list"
      value={todaysTheme}
      onChange={(e) => handleChange(e.target.value)}
      className="bg-none bg-transparent text-yellow-500 font-bold"
    >
      {themeList.map((theme) => {
        return (
          <option key={theme} value={theme}>
            {theme}
          </option>
        );
      })}
    </select>
  );
}
