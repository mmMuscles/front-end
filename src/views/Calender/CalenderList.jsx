import React from "react";
import { useState } from "react/cjs/react.development";

const themeList = [
  "Rest",
  "Legs",
  "Arms",
  "Cardio",
  "Core",
  "Strength",
  "Yoga",
  "Stretching",
];
export default function CalenderList() {
  return (
    <>
      <select className="dropdown">
        {themeList.map((theme) => {
          return (
            <option key={theme} value={theme}>
              {theme}
            </option>
          );
        })}
      </select>
    </>
  );
}
