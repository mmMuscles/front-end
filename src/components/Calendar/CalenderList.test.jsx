import { render } from '@testing-library/react';
import CalenderList from './CalenderList';

describe('CalenderList', () => {
    it('renders a dropdown of themes', () => {
        const { container } = render(<CalenderList todaysTheme="Rest" />);
        expect(container).toMatchSnapshot();
    })
})