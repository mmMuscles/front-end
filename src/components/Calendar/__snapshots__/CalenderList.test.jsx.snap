// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test renders the top nav header 1`] = `
<div>
  <select
    class="bg-none bg-transparent text-yellow-500 font-bold"
  >
    <option
      value="Rest"
    >
      Rest
    </option>
    <option
      value="Abs"
    >
      Abs
    </option>
    <option
      value="Arms"
    >
      Arms
    </option>
    <option
      value="Back"
    >
      Back
    </option>
    <option
      value="Calves"
    >
      Calves
    </option>
    <option
      value="Chest"
    >
      Chest
    </option>
    <option
      value="Legs"
    >
      Legs
    </option>
    <option
      value="Shoulders"
    >
      Shoulders
    </option>
    <option
      value="Yoga"
    >
      Yoga
    </option>
    <option
      value="Stretch"
    >
      Stretch
    </option>
  </select>
</div>
`;
